src/olympia/abuse/models.py

```diff
@@ -117,6 +117,7 @@ class AbuseReport(ModelBase):
         ('UNINSTALL', 1, 'Uninstall'),
         ('MENU', 2, 'Menu'),
         ('TOOLBAR_CONTEXT_MENU', 3, 'Toolbar context menu'),
+        ('AMO', 4, 'AMO'),
```

Comment:
I wonder if this should rather be "addons.mozilla.org" - but that representation is only used internally right?

Reply:
Yeah it's only for people triaging the abuse reports or reviewers, they should know...